app/models/language.rb

```diff
@@ -2,6 +2,9 @@ class Language < ActiveRecord::Base
   has_many :percent_languages
   has_many :percentages, :through => :percent_languages
 
+  has_many :language_offers
+  has_many :schools, :through => :language_offers
```

Comment:
Options on new lines:
```
has_many :schools,
  through: :language_offers
```
Also, try to use the Ruby 1.9+ syntax instead of hash rockets